pkg/handlers/embedded_cluster_node_join_command.go

```diff
@@ -20,18 +20,12 @@ type GenerateEmbeddedClusterNodeJoinCommandResponse struct {
 }
 
 type GetEmbeddedClusterNodeJoinCommandResponse struct {
-	ClusterID                 string                 `json:"clusterID"`
-	K0sJoinCommand            string                 `json:"k0sJoinCommand"`
-	K0sToken                  string                 `json:"k0sToken"`
-	K0sUnsupportedOverrides   string                 `json:"k0sUnsupportedOverrides"`
-	EndUserK0sConfigOverrides string                 `json:"endUserK0sConfigOverrides"`
-	MetricsBaseURL            string                 `json:"metricsBaseURL"`
-	EmbeddedClusterVersion    string                 `json:"embeddedClusterVersion"`
-	AirgapRegistryAddress     string                 `json:"airgapRegistryAddress"`
-	IsAirgap                  bool                   `json:"isAirgap"`
-	Proxy                     *ecv1beta1.ProxySpec   `json:"proxy,omitempty"`
-	Network                   *ecv1beta1.NetworkSpec `json:"network,omitempty"`
-	LocalArtifactMirrorPort   int                    `json:"localArtifactMirrorPort,omitempty"`
+	ClusterID              string                     `json:"clusterID"`
+	K0sJoinCommand         string                     `json:"k0sJoinCommand"`
+	K0sToken               string                     `json:"k0sToken"`
+	EmbeddedClusterVersion string                     `json:"embeddedClusterVersion"`
+	AirgapRegistryAddress  string                     `json:"airgapRegistryAddress"`
+	Spec                   ecv1beta1.InstallationSpec `json:"installationSpec,omitempty"`
```

Comment:
why not call it InstallationSpec

Reply:
I mean sure
it doesn't change anything, it's already encoded as 'installationSpec' 😆

Reply:
updated now